cmd/signing_ethereum.go

```diff
@@ -181,6 +174,37 @@ func EthereumVerify(c *cli.Context) error {
 		return fmt.Errorf("failed to unmarshal ethereum signature: [%v]", err)
 	}
 
+	err = verify(ethereumSignature)
+	if err != nil {
+		return fmt.Errorf("signature verification failed: [%v]", err)
+	}
+
+	fmt.Printf(
+		"signature verified correctly, message [%s] was signed by [%s]\n",
+		ethereumSignature.Message,
+		ethereumSignature.Address.Hex(),
+	)
+
+	return nil
+}
+
+func sign(ethereumKey *keystore.Key, message string) (*EthereumSignature, error) {
```

Comment:
Just a thought.. I noticed `go-ethereum` has functions [around signatures](https://github.com/ethereum/go-ethereum/blob/v1.9.10/accounts/keystore/keystore.go#L292). Do you think it would be useful to use it here and maybe end up with less code?

Reply:
We're using go-ethereum to produce a signature, two functions:
https://github.com/keep-network/keep-ecdsa/blob/529fc792f24554d46542beb61562b1deac530067/cmd/signing_ethereum.go#L192-L194

I was looking for one function that we could use to handle everything but it would require too much effort and we would end up with even more code and complications.